tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -547,6 +559,8 @@ stages:
             targetType: 'inline'
             workingDirectory: ${{ variables.testWorkspace }}/node_modules/${{ variables.testPackage }}
             script: |
+              # Note: we explicitly do not use 'set -eux -o pipefail' here because we want to run both sets of tests
```

Comment:
This script is already aware that it runs multiple things (two sets of tests) and that it wants to run to completion even if the first bits fail. It accounts for that, and fails the task in the end in that case, even if the second set of tests succeeds.